networkx/algorithms/tests/test_walks.py

```diff
@@ -0,0 +1,103 @@
+"""Unit tests for the :mod:`networkx.algorithms.walks` module."""
+
+from functools import partial
+
+from networkx import (
+    DiGraph,
+    all_pairs_number_of_walks,
+    cycle_graph,
+    empty_graph,
+    number_of_walks,
+    relabel_nodes,
+    single_source_number_of_walks,
+)
+from networkx.utils import pairwise
+
+
+class NumberOfWalks:
```

Comment:
I think the class names should start with "Test" for pytest to identify it as a class containing tests to run. Try
```python
class TestNumberOfWalks:
```
and see if that does the trick.

Reply:
Thanks, it is working now 👍

Reply:
All tests I wrote are passing locally, but they are all failing here 😞